src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/integration/authorization/SingleReferralEndpoints.kt

```diff
@@ -51,7 +51,7 @@ class SingleReferralEndpoints : IntegrationTestBase() {
     return tokenFactory.createEncodedToken(userID = user.id, userName = user.userName, authSource = user.authSource)
   }
 
-  @ParameterizedTest
+  @ParameterizedTest(name = "{displayName} ({argumentsWithNames})")
```

Comment:
please can we split these changes into their own commit? 

the same goes for refactoring of the readme, and other logically seperate changes.

Reply:
Sure, it's a habit of squashing everything. I'll be more vigilant in future!